src/bot/botutils.jl

```diff
@@ -18,7 +16,8 @@ end
 println(timesum(data, :ms))
 ```
 
-## Manual Benchmark (withtout using [`@snoopi_bench`](@ref))
```

Comment:
Probably we can remove this documentation about manual benchmarking. 🤔 `snoop_bench` already automates this.